Task 5/middlewares/passportFacebookMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import passport from 'passport';
+import FacebookStrategy from 'passport-facebook';
+import _ from 'underscore';
+
+const userData = [{facebookId: "1",
+               firstName: "Yahor",
```

Comment:
Consider installing an eslinter since it's getting harder to read through all these indentations)